src/queryString/getQuery.js

```diff
@@ -4,7 +4,7 @@ define(function () {
      * Gets full query as string with all special chars decoded.
      */
     function getQuery(url) {
-        url = url.replace(/#.*/, ''); //removes hash (to avoid getting hash query)
+        url = url.replace(/#.*\?/, '?'); //removes hash (to avoid getting hash query)
```

Comment:
The correct is query to come before the hash, but, on a SPA, a URL with query string before it's weird:

https://example.com/#/users/
https://example.com/#/users/?page=2

The solution to my case can be:
https://example.com/#/users/page-2/

--

But I wonder if the case should be valid only "?bar=true" or "?bar=true&ipsum=dolor"
http://example.com/foo?bar=true#lorem?ipsum=dolor